integral_view/views/zfs_management.py

```diff
@@ -976,6 +981,170 @@ def update_zfs_dataset(request):
         return_dict['base_template'] = "storage_base.html"
         return_dict["page_title"] = 'Modify ZFS dataset properties'
         return_dict['tab'] = 'view_zfs_pools_tab'
+        return_dict["error"] = 'Error modifying ZFS dataset properties'
+        return_dict["error_details"] = str(e)
+        return django.shortcuts.render_to_response("logged_in_error.html", return_dict, context_instance=django.template.context.RequestContext(request))
+
+
+def update_zfs_dataset_advanced_properties(request):
+    return_dict = {}
+    try:
+        name, err = django_utils.get_request_parameter_value(request, 'name')
```

Comment:
I think you are probably amending this PR to keep up with the change in `django_utils.get_request_parameter_values`. I'll hold until you are done with it.